gufe/components/proteincomponent.py

```diff
@@ -51,8 +51,8 @@
 }
 
 
-negative_ions = ["F", "CL", "Br", "I"]
-positive_ions = ["NA", "MG", "ZN"]
+negative_ions = ["F", "CL", "BR", "I"]
+positive_ions = ["NA", "MG", "CA", "ZN"]
```

Comment:
Would it make sense to just add every mono and divalent ions that are defined in the AMBER ffxml?

Reply:
We're probably replacing this entirely soon, so a small fix is ok for now. I'd be more interested in a test to encode the fix/behaviour